RestImageResize.EpiServer/RestImageResize.EPiServer.nuspec

```diff
@@ -1,26 +1,25 @@
 <?xml version="1.0"?>
-<package >
+<package>
   <metadata>
-    <id>RestImageResize.EPiServer</id>
-    <version>1.0.0</version>
-    <title>RestImageResize.EPiServer</title>
-    <authors>Roman Mironets, Creuna Kharkiv office</authors>
-    <owners>Roman Mironets, Creuna Kharkiv office</owners>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>RestImageResize for EPiServer 6/7/8+</title>
+    <authors>$author$</authors>
     <projectUrl>https://github.com/Romanets/RestImageResize</projectUrl>
     <iconUrl>https://raw.github.com/VladimirLevchuk/EPiProperties/master/EPiProperties/images/creuna.gif</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>
-      Includes required dependences and source code for easily start to use RestImageResize package in EPiServer CMS site.
-    </description>
-    <releaseNotes>Inital commit.</releaseNotes>
-    <copyright>Copyright 2013</copyright>
+    <description>$description$</description>
+    <releaseNotes>- Added support for EPiServer 7.5/8+</releaseNotes>
+    <copyright>Copyright © Roman Mironets 2015</copyright>
     <tags>Creuna Image Resize Transform OpenWaves ASP.NET EPiServer</tags>
     <dependencies>
+      <dependency id="EPiServer.CMS.Core" version="7.0.6" />
       <dependency id="RestImageResize" version="1.0.3" />
-      <dependency id="OpenWaves.ImageTransformations.EPiServer" version="1.0.85.251"/>
+      <dependency id="OpenWaves" version="1.0.85.259" />
+      <dependency id="OpenWaves.ImageTransformations.EPiServer" version="1.0.85.251" />
     </dependencies>
   </metadata>
   <files>
-    <file src="**.pp" target="content"/>
+    <file src="readme.txt" target="" />
```

Comment:
https://docs.nuget.org/create/creating-and-publishing-a-package#automatically-displaying-a-readme.txt-file-during-package-installation